README.md

```diff
@@ -39,30 +99,15 @@ Thus, our data layer has to rely only on the data emitted by the EVM. It cannot
 
 More components, e.g. a watchtower to challenge invalid state transitions, will be added in the future.
 
-## Getting Started
-
-### Requirements
+### Running the driver/listener
 
-- Rust (stable)
-- NodeJS 11.0, starting with version 12 some deprecated APIs were removed that cause `scrypt`, `keccak`, `secp256k1`, and `sha3` packages to fail to build
-- Docker and Docker-compose (stable)
-
-The project may work with other versions of these tools but they are not tested.
-
-### Installation
-
-Clone the repository, its submodule, and run the container
-```bash
-git clone git@github.com:gnosis/dex-services.git
-cd dex-services
-git submodule update --init
-docker-compose up -d ganache-cli
-(cd dex-contracts && yarn && npx truffle migrate && npm run networks-inject)
-docker-compose up
+```
+docker-compose down && docker-compose up driver graph-listener truffle
 ```
 
 This will start:
 ganache-cli, the local ethereum chain
+a truffle image compiling and deploying your smart contracts to the local chain
 postgres, the database storing the data of the snapp
 graph-listener, a listener pulling data from the ganache-cli and inserting it into postgres
 driver, a service calculating the new states and push these into the smart contract
```

Comment:
Can we itemize or enumerate this list... I found the transaction from one line to the next confusing before I realized it was bullet points.